web/src/components/LandingPageSlider.tsx

```diff
@@ -99,65 +94,22 @@ export const LandingPageSlider = (): ReactElement => {
-          tileText2={Config.landingPage.landingPageTile1Line2Text}
-          dataTestId={"get-started-tile"}
-          reference={cardRef}
-          onClick={routeToOnboarding}
-          isActive={activeTile === 0}
-          isPrimary
-        />
-        <LandingPageActionTile
-          className={"landing-page-slide"}
-          imgPath={"/img/briefcase-icon.svg"}
-          dataTestId={"register-biz-tile"}
-          tileText={Config.landingPage.landingPageTile2Text}
-          onClick={(): void => setFlowAndRouteUser("starting")}
-          isActive={activeTile === 1}
-        />
-        <LandingPageActionTile
-          className={"landing-page-slide"}
-          imgPath={"/img/payTaxes-icon.svg"}
-          dataTestId={"pay-taxes-tile"}
-          tileText={Config.landingPage.landingPageTile3Text}
-          onClick={(): void => setFlowAndRouteUser("up-and-running")}
-          isActive={activeTile === 2}
-        />
-        <LandingPageActionTile
-          className={"landing-page-slide"}
-          imgPath={"/img/outOfState-icon.svg"}
-          dataTestId={"out-of-state-tile"}
-          tileText={Config.landingPage.landingPageTile4Text}
-          onClick={(): void => setFlowAndRouteUser("out-of-state")}
-          isActive={activeTile === 3}
-        />
-        <LandingPageActionTile
-          className={"landing-page-slide"}
-          imgPath={"/img/eligibleFunding-icon.svg"}
-          dataTestId={"eligible-funding-tile"}
-          tileText={Config.landingPage.landingPageTile5Text}
-          onClick={(): void => setFlowAndRouteUser("up-and-running")}
-          isActive={activeTile === 4}
-        />
-        <LandingPageActionTile
-          className={"landing-page-slide"}
-          imgPath={"/img/startBusiness-icon.svg"}
-          tileText={Config.landingPage.landingPageTile6Text}
-          dataTestId={"start-biz-tile"}
-          onClick={(): void => setFlowAndRouteUser("starting")}
-          isActive={activeTile === 5}
-        />
-        <LandingPageActionTile
-          className={"landing-page-slide"}
-          imgPath={"/img/runBusiness-icon.svg"}
-          tileText={Config.landingPage.landingPageTile7Text}
-          dataTestId={"run-biz-tile"}
-          onClick={(): void => setFlowAndRouteUser("up-and-running")}
-          isActive={activeTile === 6}
-        />
+        {props.actionTileProps.map((actionTile, index) => {
+          return (
+            <LandingPageActionTile
+              key={`landing-page-slider-key-${index}`}
+              className={"landing-page-slide"}
+              imgPath={actionTile.imgPath}
+              dataTestId={actionTile.dataTestId}
+              tileText={actionTile.tileText}
+              tileText2={actionTile.tileText2}
+              onClick={actionTile.onClick}
+              isPrimary={actionTile.isPrimary}
+              isActive={activeTile === index}
+              reference={index === 0 ? cardRef : undefined}
```

Comment:
Per our convo about making the ActionTile type shared, I would also change `LandingPageActionTile` to take this same type as a prop, instead of having to split it out into so many individual variables here.